### ISSUE TITLE
pulumi plugin install tries to install plugin that is not plugin

### ISSUE THREAD

**Issue by jancespivo** [pulumi/pulumi#8011]
Steps to reproduce:

`pip install pulumi-tools`

It is completely empty project I've created.

`pulumi plugin install`

throws an error:

```
[resource plugin tools-0.0.1] installing
error: [resource plugin tools-0.0.1] downloading from : 403 HTTP error fetching plugin from https://get.pulumi.com/releases/plugins/pulumi-resource-tools-v0.0.1-linux-amd64.tar.gz
```

It probably looks for all packages with a prefix `pulumi-` regardless they are providers or not.

**Comment by emiliza** [comment]
Looks like this is getting picked up as a ResourcePlugin. I anticipate we may fix this as part of #6932

**Comment by justinvp** [comment]
I'm going to close this as no longer relevant. `pulumi-tools` hasn't been updated since Sep 16, 2021.

By default, Pulumi will try to install plugins for packages prefixed with `pulumi-` or `pulumi_`. If your package doesn't have an associated plugin, you can add a `pulumi-plugin.json` file to your PyPi package with the following content:

```json
{ "resource": false }
```

See https://github.com/pulumi/pulumi-policy/blob/cce44759b90ded3b7aba6b0bfa8f30df1297a11c/sdk/python/lib/pulumi_policy/pulumi-plugin.json
